src/app/haku/[oid]/yhteisvalinnan-hallinta/page.tsx

```diff
@@ -6,20 +6,22 @@ import { ExternalLink } from '@/components/external-link';
 import { LabeledInfoItem } from '@/components/labeled-info-item';
 import { QuerySuspenseBoundary } from '@/components/query-suspense-boundary';
 import { buildLinkToHaku } from '@/lib/ataru/ataru-service';
-import { configuration } from '@/lib/configuration';
 import { useHaku } from '@/lib/kouta/useHaku';
 import { useTranslations } from '@/lib/localization/useTranslations';
 
 import { Stack } from '@mui/material';
 import { OphLink } from '@opetushallitus/oph-design-system';
-import { use } from 'react';
+import { use, useContext } from 'react';
 import { YhteisvalinnanValintalaskenta } from './components/yhteisvalinnan-valintalaskenta';
 import { SijoitteluContainer } from './components/sijoittelu-container';
 import { LettersContainer } from './components/letters-container';
+import { ConfigurationContext } from '@/components/providers/configuration-provider';
 
 const YhteisvalinnanHallintaContent = ({ hakuOid }: { hakuOid: string }) => {
   const { data: haku } = useHaku({ hakuOid });
 
+  const { configuration } = useContext(ConfigurationContext);
```

Comment:
https://react.dev/blog/2024/12/05/react-19#new-feature-use

```suggestion
  const { configuration } = use(ConfigurationContext);
```